docs/reference/cli-reference/cli-ref-search.md

```diff
@@ -0,0 +1,65 @@
+---
+title: NuGet CLI search command
+description: Reference for the nuget.exe search command
+author: advay26
+ms.author: t-adtand
+ms.date: 07/07/2020
+ms.topic: reference
+---
+
+# search command (NuGet CLI)
+
+**Applies to:** package consumption &bullet; **Supported versions:** all
```

Comment:
`nuget search` is not available in all versions of nuget.exe.

Reply:
Yes, I thought as much, but wasn't sure what the right answer would be. Do you happen to know which versions would support `search`?

Reply:
NuGet.Client's dev branch is in the process of being rebranded to 5.8, but it depends on when the feature was merged into client's dev branch.

Reply:
I've changed it to TODO for now, and I'll update it based on when the command is merged with dev.